### PR TITLE
Fixes #4 - Resolve bugzilla issue as "Moved" and add a link to See also

### LINKED ISSUE #4: Resolve bugzilla issue as "Moved" and add a link to See also
Once an issue has been created on the webcompat side we need to:

- add a link to See also field of the bugzilla bug
- resolve the bug as "Moved"

### PR DESCRIPTION
**Pull request by ksy36**
In this PR I've added the following:
- once the issue has been created on webcompat.com add a link to "See also" field of the bugzilla issue 
- close the bugzilla issue as "Moved"
- improved error handling and added loading state for the button

### CODE REVIEW

File: moveToWebcompat.js [hunk 1]
```diff
@@ -86,42 +111,89 @@ const sendToWebcompat = async bug => {
...
-  if (!bugId)
-    openDropDown(
+  if (!bugId) {
+    return showError(
       "A valid bug ID could not be found. Please check you're actually viewing a bug."
     );
+  }
 
-  const result = await getBugData(bugId);
+  startLoadingState();
```
Comment (wisniewskit) [inline moveToWebcompat.js]: This function name makes it sound a little more scary than what it really is doing; maybe we should change the name of this and the stop function to something like "disableMoveButton" and "enableMoveButton"?

File: moveToWebcompat.js [hunk 2]
```diff
@@ -86,42 +111,89 @@ const sendToWebcompat = async bug => {
...
+  [...document.getElementById("resolve-as").children]
+    .filter(el => el.textContent.includes("FIXED"))[0]
+    .click();
+
+  const resolution = document.getElementById("bottom-resolution");
+  resolution.value = "MOVED";
+  resolution.dispatchEvent(new Event("change"));
 
-  return moveButton;
+  setTimeout(() => document.getElementById("bottom-save-btn").click(), 100);
```
Comment (wisniewskit) [inline moveToWebcompat.js]: Why must we use a `setTimeout` here? If it's to wait for the change event above to be handled first, it seems racy.

File: moveToWebcompat.js (hunk 1 above)
Comment (ksy36) [reply moveToWebcompat.js]: Yeah good point, thanks :)

File: moveToWebcompat.js (hunk 2 above)
Comment (ksy36) [reply moveToWebcompat.js]: Mmm it's not really needed here, didn't remove the timeout after testing. 

Initially I have been trying to change the select value by just setting `resolution.value = "MOVED";`, but it didn't get saved. I wasn't sure why, so tried to add a timeout, but it wasn't the reason. Tried dispatching change event and it worked as bugzilla's js script is expecting a change event to save the value (the keep track of "dirty" fields, and if the field hasn't been touched, they override it to be the initial value).

**Review by wisniewskit — APPROVED** [review]

**Pull request merged**